### DISCUSSION TITLE
Sendeleistung der DTU lässt sich nicht ändern

### DISCUSSION THREAD

**Discussion by schnulli0859** [tbnobody/OpenDTU#1890] [General]
Moin,
wenn ich die Sendeleistung der DTU ändern möchte kommt immer die Anzeige "Die Frequenz muss zwischen 860250000 und 923500000 kHz liegen und ein Vielfaches von 250kHz betragen!".
Dieses passiert für die gesamte Seite, also auch wenn ich die Intervallzeit ändern möchte.
Wo mache ich da einen Fehler?
Hardware ist ESP32-D0WD-V3 mit nRF24L01 und aufgeschraubter Antenne. SW ist v24.3.31.

Vielen Dank im voraus.

**Comment by tbnobody** [comment]
Bitte die Webseite mit Strg+F5 neu laden oder den Browser Cache leeren. Es scheint als hättest du von einer ziemlich alten Version auf die aktuelle upgedated.
